core/src/constants/METAMASK_ERRORS.ts

```diff
@@ -0,0 +1,6 @@
+const METAMASK_ERRORS: Record<string, string> = {
+    'Bidding error Error: cannot estimate gas; transaction may fail or may require manual gas limit (error={"code":-32603,"message":"execution reverted: Clipper/not-running-auction","data":{"originalError":{"code":3,"data":"0x08c379a00000000000000000000000000000000000000000000000000000000000000020000000000000000000000000000000000000000000000000000000000000001b436c69707065722f6e6f742d72756e6e696e672d61756374696f6e0000000000","message":"execution reverted: Clipper/not-running-auction"}}}, method="estimateGas", transaction={"from":"0xB2E1284Fb685dbDcc4378cF5d4E9515679527a42","maxPriorityFeePerGas":{"type":"BigNumber","hex":"0x59682f00"},"maxFeePerGas":{"type":"BigNumber","hex":"0xbaabef8640"},"to":"0xb0ece6F5542A4577E2f1Be491A937Ccbbec8479e","data":"0x81a794cb000000000000000000000000000000000000000000000000000000000000000200000000000000000000000000000000000000000000000ff4327f3f09430c010000000000000000000000000000000000000004aaebc81e9fe122a2aab7215400000000000000000000000074893c37beacf205507ea794470b13de0629422000000000000000000000000000000000000000000000000000000000000000a00000000000000000000000000000000000000000000000000000000000000180000000000000000000000000b2e1284fb685dbdcc4378cf5d4e9515679527a42000000000000000000000000f11a98339fe1cde648e8d1463310ce3ccc3d7cc1000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000a0000000000000000000000000000000000000000000000000000000000000012000000000000000000000000000000000000000000000000000000000000000030000000000000000000000001f9840a85d5af5bf1d1762f925bdaddc4201f984000000000000000000000000c02aaa39b223fe8d0a0e5c4f27ead9083c756cc20000000000000000000000006b175474e89094c44da98b954eedeac495271d0f0000000000000000000000000000000000000000000000000000000000000002000000000000000000000000c02aaa39b223fe8d0a0e5c4f27ead9083c756cc20000000000000000000000006b175474e89094c44da98b954eedeac495271d0f","type":2,"accessList":null}, code=UNPREDICTABLE_GAS_LIMIT, version=providers/5.4.5)':
```

Comment:
- [ ] Let's not store it as part of our core, but only as part of our frontend storybook file
- [ ] The error thrown [here](https://github.com/sidestream-tech/unified-auctions-ui/blob/b0970c58b7258335f59403194821bbadd621339b/frontend/store/auctions.ts#L207) so the actual error message thrown by the metamask starts after `Error: ` – this is what you need to give as input to the function
- [ ] no need to store output here